sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/io/BigQueryIO.java

```diff
@@ -24,6 +24,7 @@
 import com.google.api.services.bigquery.model.TableSchema;
```

Comment:
I think we should have a CustomSource/CustomSink tester.
individual methods in writer and write operation are very simple. I don't see value to test them out side the CustomIO frame work.

I think end-to-end BigQueryIOTest should give us enough coverage for this CL.